std_msgs/Header header
string obj_class
sensor_msgs/PointCloud2 cloud